src/nix/flake.md

```diff
@@ -415,6 +420,244 @@ way. Most flakes provide their functionality through Nixpkgs overlays
 or NixOS modules, which are composed into the top-level flake's
 `nixpkgs` input; so their own `nixpkgs` input is usually irrelevant.
 
+## Flake outputs
+
+While the technical specification of a flake does not include any details about
+the outputs, there are some common conventions, and tools that expect certain
+outputs. These are documented here, including how they are typically used,
+whether they are recognized by [`nix flake check`](./flake-check.md), and what
+type they should be.
+
+### checks
+Used-by: `nix flake check`
+Checked: Yes
+Type: systemAttrs (attrsOf (derivation))
```

Comment:
This doesn't format in a nice way, namely "Used-by: nix flake check Checked: Yes Type: systemAttrs (attrsOf (derivation))".